src/main/java/io/vertx/mqtt/MqttClientOptions.java

```diff
@@ -230,6 +232,17 @@ public boolean isAutoGeneratedClientId() {
     return isAutoGeneratedClientId;
   }
 
+  public int getMaxMessageSize() {
+    return maxMessageSize;
+  }
+
+  public MqttClientOptions setMaxMessageSize(int maxMessageSize) {
+    Arguments.require(maxMessageSize > 0 , "maxMessageSize must be > 0");
+    setReceiveBufferSize(maxMessageSize);
+    this.maxMessageSize = maxMessageSize;
+    return this;
+  }
+
```

Comment:
But I m just a bit disappointed by the fact we have two methods for limiting the incoming messages size: setMaxMessageSize and setReceiveBufferSize from extended class.

So in case, we want to limit incoming message size should we call two methods for that purpose?  Can we better call only one method instead?